samza-core/src/main/scala/org/apache/samza/config/MetricsConfig.scala

```diff
@@ -20,16 +20,18 @@
 package org.apache.samza.config
 
 
-import org.apache.samza.util.HighResolutionClock
+import org.apache.samza.system.SystemStream
 
 
 object MetricsConfig {
   // metrics config constants
   val METRICS_REPORTERS = "metrics.reporters"
   val METRICS_REPORTER_FACTORY = "metrics.reporter.%s.class"
   val METRICS_SNAPSHOT_REPORTER_STREAM = "metrics.reporter.%s.stream"
-  val METRICS_SNAPSHOT_REPORTER_INTERVAL= "metrics.reporter.%s.interval"
-  val METRICS_TIMER_ENABLED= "metrics.timer.enabled"
+  val METRICS_SNAPSHOT_REPORTER_INTERVAL = "metrics.reporter.%s.interval"
+  val METRICS_TIMER_ENABLED = "metrics.timer.enabled"
+  val METRICS_SNAPSHOT_REPORTER_PARTITION_KEY = "metrics.reporter.%s.partitionkey"
```

Comment:
Just curious, shouldn't we have a default value for this configuration variable?